draft-ietf-httpbis-unprompted-auth.md

```diff
@@ -232,6 +232,10 @@ the browser such that it could access another context's memory, the attacker
 might also be able to access the corresponding key, so binding authentication to
 requests would not provide much benefit in practice.
 
+Key material used for authentication in unprompted auth, whether symmetric or
```

Comment:
Please don't use "unprompted auth", that abbreviation is not used in the document

Reply:
Additionally, keys are used for the new authentication schemes more than unprompted auth so we should just mention those instead.